src/drivers/mct/cime_config/config_component_cesm.xml

```diff
@@ -291,7 +288,7 @@
 
   <entry id="GLC_NCPL">
     <type>integer</type>
-    <default_value>1</default_value>
+    <default_value>$ATM_NCPL</default_value>
```

Comment:
This will change coupling frequency for cases with active land and CISM, which we don't want to do.